clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -2800,6 +2800,11 @@ private int internalProcessConsumerRecord(
         }
         if (recordLevelMetricEnabled.get()) {
           versionedDIVStats.recordSuccessMsg(storeName, versionNumber);
+          versionedDIVStats.recordDrainerDIVCompletionTime(
+              storeName,
+              versionNumber,
+              LatencyUtils.getElapsedTimeInMs(currentTimeMs),
+              currentTimeMs);
```

Comment:
Is this useful? We used to have a DIV latency metric and we removed it since a profiling showed that DIV itself was taking less CPU time than the metric recording time... In other words... it was such a vanishingly small number that it did not matter to instrument it. Unless we're planning to add significant complexity to the DIV logic and fear that its runtime will increase significantly, then I would like to avoid adding one more hot path metric, of which we already have too many...

Reply:
@huangminchn recently had some investigation and suspect the DIV in drainer might be the bottleneck and that's why we are adding this here. 
@huangminchn  can you please chime in and see if it's necessary?

Reply:
Yes, our investigation showed that the drainer spent ~4+ ms to process one single message in one of the certification chaos test; rocksDB put only took a few micro seconds, and there was not too many rocksDB compaction happening; GC looked good. The only bottleneck I could think of is DIV processing; I would like to confirm this.

Reply:
I don't understand... does this metric measure just the DIV execution time? Or does it measure some other scope of work (of which the last step is DIV completion, perhaps?)...? If it is the former, then I don't believe there is value in that metric, as I stated above. If it is the latter, that may be fine, but then we should rename the metric to more accurately match what is it that it is measuring.

Reply:
That's the latter actually! The naming isn't very good.. I will rename them!